src/workerd/api/node/tests/zlib-nodejs-test.js

```diff
@@ -1,12 +1,11 @@
-import {
-  strictEqual,
-  throws,
-  deepStrictEqual,
-  ok as assert,
-} from 'node:assert';
+import assert from 'node:assert';
 import { Buffer } from 'node:buffer';
+import { Readable } from 'node:stream';
+import { inspect, promisify } from 'node:util';
 import zlib from 'node:zlib';
 
+const { strictEqual, throws, deepStrictEqual } = assert;
```

Comment:
If we are making this distinction let's make everything `assert.fn` and let's remove this line